docs/diagrams/UiClassDiagram.puml

```diff
@@ -40,6 +41,8 @@ MainWindow *-down-> ArchiveModeBox
 
 PersonListPanel -down-> PersonCard
 
+PersonCard -down-> ColorPicker
```

Comment:
Perhaps this should be a dependency (dashed line) instead of association

Reply:
oh true

Reply:
I think the diagram wasnt saved, can you check again?

Reply:
For the delete sequence diagram, I think the extra X at the DeleteCommand wasn't saved as well. 
Can you also update the tip to  **Note:** The lifeline for `DeleteCommandParser` and `DeleteCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.

Reply:
O.OOOOO Ok Ill check